app/src/main/java/space/taran/arknavigator/utils/Tags.kt

```diff
@@ -20,7 +20,7 @@ class Converters {
             .split(',')
             .map { it.trim() }
             .filter {
-                if (it.isEmpty()) throw AssertionError("No tag can be empty")
+                if (it.isEmpty()) return@filter false
```

Comment:
I don't think so. As you can see, instead of throwing an exception, we return false now. That means, that filtering won't accept this file, because it doesn't reach a "return true" statement.